Description: Remove duplicate RPATH settings in CMakeLists.txt
 All RPATH settings are set in each plugin separately, so these extra
 settings produce 2 RPATH entries.
Author: James Cowgill <james410@cowgill.org.uk>
Forwarded: http://forums.codelite.org/viewtopic.php?f=13&t=2506
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -410,16 +410,6 @@ if ( APPLE )
     execute_process(COMMAND cp ${CMAKE_SOURCE_DIR}/Runtime/make_mac_bundle.sh pack)
 endif( APPLE )
 
-###########################################
-## RPATH settings
-###########################################
-
-if (UNIX AND NOT APPLE)
-    SET(CMAKE_INSTALL_RPATH ${PLUGINS_DIR})
-    SET(CMAKE_SKIP_BUILD_RPATH  FALSE)
-    SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-endif(UNIX AND NOT APPLE)
-
 if ( WITH_PCH )
     set( USE_PCH 1 )
 endif( WITH_PCH )
